luigi/scheduler.py

```diff
@@ -77,6 +79,38 @@ class Scheduler(object):
 
 TASK_FAMILY_RE = re.compile(r'([^(_]+)[(_]')
 
+RPC_METHODS = {}
+
+
+def rpc_method(fn=None, **request_args):
+
+    # If request args are passed, return this function again for use as
+    # the decorator function with the request args attached.
+    if fn is None:
+        return functools.partial(rpc_method, **request_args)
```

Comment:
Can we avoid this magic if we just force everyone to do `@rpc_method()` instead of the slightly more convenient `rpc_method`?